aspnetcore/signalr/client-features.md

```diff
@@ -16,9 +16,15 @@ The table below shows the features and support for the clients that offer real-t
 
 | Feature | .NET Core | JavaScript | Java |
 | ---- | :-: | :-: | :-: |
+| Azure SignalR Service Support |✔|✔|✔|
 | [Server-to-client Streaming](xref:signalr/streaming)          |✔|✔|✔|
 | [Client-to-server Streaming](xref:signalr/streaming)          |✔|✔|✔|
 | Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |✔|✔| |
+| WebSockets Transport |✔|✔|✔|
+| Server-Sent Events Transport |✔|✔| |
```

Comment:
FYI, we have no plans to support SSE in Java (and are exploring ways to phase it out of all clients), so I didn't reference an issue for it in the list below.